pydsa/dfs.py

```diff
@@ -0,0 +1,16 @@
+# Depth First Search
+# Complexity: O(V+E)
+#uses concept of list
+
+def dfsearch(graph,s):
```

Comment:
definitely, function name must be DFS but then it doesn't work. import module error arise which i discussed. I don't know why BFS is working. i tried creating it as much as similar to BFS as it can be. Will look again at it.